### PR TITLE
First try with nuget. Please just merger after a clearly look

### PR DESCRIPTION
**Pull request by frenzqse**

### CODE REVIEW

File: bridge/Implementation/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -5,12 +5,12 @@
...
-[assembly: AssemblyDescription("")]
+[assembly: AssemblyTitle("BridgeImplementation")]
+[assembly: AssemblyDescription("Implementation of the .Net Bridge Implemenation")]
 [assembly: AssemblyConfiguration("")]
-[assembly: AssemblyCompany("Microsoft")]
-[assembly: AssemblyProduct("RealDomainService")]
-[assembly: AssemblyCopyright("Copyright © Microsoft 2011")]
+[assembly: AssemblyCompany("OpenEngSB")]
+[assembly: AssemblyProduct("Implemenation")]
+[assembly: AssemblyCopyright("Copyright © OpenEngSB 2013")]
```
Comment (anpieber) [inline bridge/Implementation/Properties/AssemblyInfo.cs:13]: couldn't we use a $year property or something similar here? Just afraid that I'll forget to upgrade this 2014 :-)

File: bridge/Implementation/Properties/AssemblyInfo.cs [hunk 2]
```diff
@@ -32,6 +32,6 @@
 // You can specify all the values or you can default the Build and Revision Numbers 
 // by using the '*' as shown below:
 // [assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyVersion("1.0.0")]
-[assembly: AssemblyFileVersion("1.0.0")]
+[assembly: AssemblyVersion("1.0.0.0")]
```
Comment (anpieber) [inline bridge/Implementation/Properties/AssemblyInfo.cs:35]: there's no chance to do this only in one place for all projects in the bridge, is there?

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java [hunk 3]
```diff
@@ -7,6 +7,8 @@
 import java.io.FileWriter;
```
Comment (anpieber) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:7]: I've thought we've removed all java files?

**Comment by anpieber** [comment]
except for my comments all I'm missing is a readme file tbh; otherwise nice work! thanks

File: wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java (hunk 3 above)
Comment (frenzqse) [inline wsdltodll/src/main/java/org/openengsb/loom/csharp/comon/wsdltodll/WsdlToDll.java:7]: I removed it in the new pull request. I first fixed the bug from the plugin before I removed everything.

File: bridge/Implementation/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (frenzqse) [inline bridge/Implementation/Properties/AssemblyInfo.cs:13]: Unfrotunatly not :-(
http://stackoverflow.com/questions/1979873/current-year-in-assemblyinfo-copyright

File: bridge/Implementation/Properties/AssemblyInfo.cs (hunk 2 above)
Comment (frenzqse) [inline bridge/Implementation/Properties/AssemblyInfo.cs:35]: This one was wrong. The correct one is in the new Pull request (nuget)

**Comment by frenzqse** [comment]
The readme file is in the new Pull request (Nuget)

**Pull request closed without merging**